README.md

```diff
@@ -211,6 +211,23 @@ archive { '/nfs/repo/software.zip':
 }
 ```
 
+### Extract Customization
+
+The `extract_flag` or `extract_command` parameters can be used to override the
```

Comment:
Can you give an example of extract_flag as well, or is it a simple alias? If the latter maybe a trailing comment on line 224 saying '# you can also use extract_flag" or similar.